README.md

```diff
@@ -1 +1,43 @@
-# css
\ No newline at end of file
+# CSS
+
+This is the CSS package for WARP.
+
+The package includes only resets for now.
+
+## Usage
+
+The CSS should be used directly from Eik. This is because we can alias releases by major versions.
+
+We publish 2 versions of presets. The minified that can be used in production and non-minified that can be used in development.
+
+```html
+<link href="https://assets.finn.no/pkg/@warp-ds/css/v1/resets.min.css" rel="stylesheet" />
+or
+<link href="https://assets.finn.no/pkg/@warp-ds/css/v1/resets.css" rel="stylesheet" />
+```
+
+### Releases
```

Comment:
Maybe we could follow the structure of contribution docs that @magnuh has done in https://github.com/warp-ds/vue?

Reply:
Added here https://github.com/warp-ds/css/pull/2/commits/186e78394a9e5dcc28b917fe4f62c9bc222a2344